php-config/Slate/UI/SectionProfile.config.d/gsuite-tools.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+Slate\UI\SectionProfile::$sources[] = function (Slate\Courses\Section $Section) {
+    $links = [];
+
+    if (!empty($GLOBALS['Session']) && $GLOBALS['Session']->hasAccountLevel('Staff')) {
+        $sectionStudentEmails = array_filter(array_map(function($Student) {
```

Comment:
everything looks really well-done, but I wonder if now that we're creating the calendar event from the backend, we could avoid loading the list of students on every page load and instead either:

- have the JS make an XHR call to load the section roster
- have the create event call pass a section id or student list string to the create event handler